base-centos-6.9/packer.json

```diff
@@ -18,6 +18,6 @@
 
   "provisioners": [{
     "type": "shell",
-    "script": "./cloud-init-centos-6.9.sh"
+    "script": "./init.sh"
```

Comment:
I thought with making everything generic it would be good to call this simply `init.sh`.